src/pages/stocks/index.tsx

```diff
@@ -22,6 +22,25 @@ const Stocks: NextPage = () => {
         <Form.Item name="name" label="Name" rules={[{ required: true }]}>
           <Input />
         </Form.Item>
+        <Form.Item name="cost" label="Cost" rules={[{ required: true }]}>
+          <InputNumber size="large" min={0} max={100000}  defaultValue={0} style={{ width: '100%' }} />
```

Comment:
```suggestion
          <InputNumber size="large" min={0} max={100000}  defaultValue={0} style="w-full" />
```
And we shouldn't have a max value.